### PR TITLE
Company user profile

### PR DESCRIPTION
**Pull request by bago2k4**
card:
https://trello.com/c/5t4wTi4o

To test:
- [x] import blank data
- [x] open blank profile
- [x] change name, currency and description and save
- [x] change name and logo (use a valid url that doesn't point to an image, ie: _http://google.com_), you shouldn't be able to save the data, an alert should appear
- [x] change name and logo (use a valid image url)
- [x] open a company page
- [x] navigate to the user info page (from the menu)
- [x] click the "BACK TO DASHBOARD" button, you should go back to the company (this happens only if you used push state to navigate to the profile, if you reload the user profile page you should be sent back to the home page)

NB: I suggested to save the company avatar to filestack but since we have some stuff to finish before the end of the week i prefer to work on more urgent things. Added [this](https://trello.com/c/F0wNvGXp) to the Technical Debt Backlog lane of trello

### CODE REVIEW

File: scss/partials/_back_to_dashboard_btn.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    font-size: 12px;
+    line-height: 1.5;
+    letter-spacing: 1px;
+    color: rgba(78,90,107, 0.5);
+    margin: 0px;
+    padding: 0px;
+    background-color: white;
+    outline: none;
+  }
+}
```
Comment (martinklepsch) [inline scss/partials/_back_to_dashboard_btn.scss]: These styles button styles are reused across the app. I added a few styles in the `improve redirect logic` PR for this type of button that are easier to reuse: https://github.com/open-company/open-company-web/pull/147/files#diff-36a224998480a3e2239228e57a8a1e76R7

Also consider using the basscss classes `mt1` - `mt4` for margin top and the `p1` - `p4` classes for uniform paddings. 

http://www.basscss.com/#basscss-margin
http://www.basscss.com/#basscss-padding

**Comment by martinklepsch** [comment]
> change name, currency and description and save

should saving maybe close the editing view?

Otherwise working as advertised 👍

File: scss/partials/_company_profile.scss [hunk 2]
```diff
@@ -1,55 +1,197 @@
...
+          height: 30px;
+          background-color: $oc_yellow;
+          position: absolute;
+          border: none;
+          outline: none;
+          font-weight: 700;
+          font-size: 12px;
+          line-height: 1.5;
+          letter-spacing: 1px;
+          color: $oc_gray_5;
```
Comment (martinklepsch) [inline scss/partials/_company_profile.scss]: @bago2k4 this should also be replaced with `btn-solid`

**Comment by belucid** [comment]
@bago2k4 merge mainline

**Comment by belucid** [comment]
@bago2k4 Remove the description field from this UI.

**Comment by belucid** [comment]
@bago2k4 if you refresh the page while looking at the profile, all is not well.

**Comment by belucid** [comment]
@bago2k4 style the slug in grey so it looks like text on the background rather than looking like the other, editable fields. Put the server URL in front of the slug, eg.: localhost:3000/buffer  staging.opencompany.com/buffer opencompany.com/buffer

**Comment by belucid** [comment]
@bago2k4 I believe there is supposed to be some language about the user info coming from Slack and changing it there, yes?

**Comment by bago2k4** [comment]
@belucid ready again if CI is happy

**Comment by martinklepsch** [comment]
> if you refresh the page while looking at the profile, all is not well.
@bago2k4 after reloading `http://localhost:3559/buffer/profile` the data seems to be still missing

**Comment by martinklepsch** [comment]
I updated @belucid's comments which checkboxes and ticked where appropriate. Reloading the company profile still seems to not fetch the required data

**Comment by belucid** [comment]
@bago2k4  

- [ ] please merge mainline again
- [ ] if you refresh the page while looking at the profile, all is not well. http://take.ms/DzM0H
- [ ] Fix footer to the bottom of the page http://take.ms/2BN80
- [ ] Opening of drawer on big web pushes header down, then over http://take.ms/Q6SYP

**Comment by bago2k4** [comment]
@belucid ready again

**Comment by belucid** [comment]
Reviewing

**Pull request merged**